uisdk/src/test/java/com/karhoo/uisdk/screen/rides/detail/RideDetailPresenterTest.kt

```diff
@@ -74,13 +77,14 @@ class RideDetailPresenterTest {
 
     @Before
     fun setUp() {
-        KarhooUISDKConfigurationProvider.setConfig(configuration = UnitTestUISDKConfig(context =
-                                                                                       context,
-                                                                                       authenticationMethod = AuthenticationMethod.KarhooUser()))
+        KarhooUISDKConfigurationProvider.setConfig(configuration = UnitTestUISDKConfig(context = context,
```

Comment:
UnitTestUISDKConfig.setKarhooAuthentication(context) sets this without as much code

Reply:
Replaced it